tls/_common/prf.py

```diff
@@ -0,0 +1,69 @@
+# This file is dual licensed under the terms of the Apache License, Version
```

Comment:
I think that's a good idea. I added it, but I am not sure if that's a thing that sphinx understands. (Is it supposed to go before or after the imports? :| )

 I suppose a good way around this would be to do what `cryptography` does and have the documentation be separate from the actual implementation: https://github.com/pyca/cryptography/blob/master/docs/hazmat/primitives/mac/hmac.rst.
Twisted also has `howto`s separate from the API documentation.